api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -113,6 +113,38 @@ public void testCreateInstitution_withTierRequirement() {
     assertThat(service.getInstitutions()).isStrictlyOrdered(comparator);
   }
 
+  @Test
+  public void testCreateInstitutionError_accessTierNotFound() {
+    final Institution anotherInst =
+        new Institution()
+            .shortName("otherInst")
+            .displayName("An Institution for Testing")
+            .addTierConfigsItem(
+                rtTierConfig
+                    .membershipRequirement(InstitutionMembershipRequirement.DOMAINS)
+                    .eraRequired(false)
+                    .accessTierShortName("non exist tier"))
+            .organizationTypeEnum(OrganizationType.INDUSTRY);
+
+    assertThrows(NotFoundException.class, () -> service.createInstitution(anotherInst));
```

Comment:
this is a nicer pattern than encapsulating the whole test in "assertThrows()" - I'm going to start using this 👍